src/Model/Item.php

```diff
@@ -497,7 +511,7 @@ private static function isDuplicate(array $item)
 			}
 		} elseif ($item['network'] == Protocol::OSTATUS) {
 			// Check for an existing post with the same content. There seems to be a problem with OStatus.
-			$condition = ["`body` = ? AND `network` = ? AND `created` = ? AND `contact-id` = ? AND `uid` = ?",
+			$condition = ['`body` = ? AND `network` = ? AND `created` = ? AND `contact-id` = ? AND `uid` = ?',
```

Comment:
```suggestion
			$condition = ["`body` = ? AND `network` = ? AND `created` = ? AND `contact-id` = ? AND `uid` = ?",
```

Reply:
Do you want me to fix the indenting here?

Reply:
No, it's fine, the values are following the SQL snippet order, it's better this way.